### ISSUE TITLE
inlets/outlets do not hide when changing from run/edit mode for ELSE GUI objects

### ISSUE THREAD

**Issue by porres** [timothyschoen/PlugData#1907]
For ELSE objects, the logic is the same from MAX, where inlets get hidden when in run mode and only show if you're in edit mode. 

This is I guess the only design choice I liked and stole from MAX :) 

I know it is inconsistent to iemguis, so I don't know how you feel about that. 

Would you do that even though it is inconsistent to iemguis? That's what I'm kinda asking... but there are other options.

Maybe you just don't do that or anything and just keep things consistent as they are?

Maybe you like this idea too and apply the same MAX principle for ELSE/iemguis/atom boxes? I'd be cool with that :)  

Or maybe you like the idea of hiding inlets/outlets all the time while in run mode and you can add this as some sort of setting?

**Comment by timothyschoen** [comment]
> For ELSE objects, the logic is the same from MAX, where inlets get hidden when in run mode and only show if you're in edit mode.

From the perspective of a plugdata user, wouldn't it make more sense if this was consistent between all objects?

**Comment by porres** [comment]
> From the perspective of a plugdata user, wouldn't it make more sense if this was consistent between all objects?

yes, this is why I said things like "Maybe you just don't do that or anything and just keep things consistent as they are?"

Well, anyway, not sure what you mean here, or what's your choice. One option I gave is that you could maybe adopt MAX's design option, which I think is good and you do get much inspiration from that as well.

**Comment by timothyschoen** [comment]
I'm leaving this as is, to keep consistency between all objects within plugdata
